tests/foreman/cli/test_scparam.py

```diff
@@ -1,47 +0,0 @@
-# -*- encoding: utf-8 -*-
-"""Test class for Smart Class Parameter CLI.
```

Comment:
That module is duplication of `test_classparameters.py`, so removing in favor of it, because it contains much more test cases